src/api_v1.py

```diff
@@ -8,11 +8,12 @@
 
 import src.config as config
 from src.exceptions import HTTPError
+from fabric8a_auth.auth import login_required
 from src.trained_model_details import trained_model_details
 from rudra.utils.validation import check_field_exists
-from rudra.deployments.emrs.pypi_emr import PyPiEMR
-from rudra.deployments.emrs.maven_emr import MavenEMR
-from rudra.deployments.emrs.npm_emr import NpmEMR
+from rudra.deployments.emr_scripts.pypi_emr import PyPiEMR
+from rudra.deployments.emr_scripts.maven_emr import MavenEMR
+from rudra.deployments.emr_scripts.npm_emr import NpmEMR
 
 
 daiquiri.setup(level=logging.DEBUG)
```

Comment:
remove hard code debug level and use environment variable
https://github.com/fabric8-analytics/fabric8-analytics-rudra/blob/master/rudra/__init__.py#L8
https://github.com/fabric8-analytics/fabric8-analytics-rudra/blob/master/rudra/__init__.py#L15